Library/Formula/aria2.rb

```diff
@@ -31,8 +32,8 @@ def install
     args = %W[
       --disable-dependency-tracking
       --prefix=#{prefix}
-      --with-appletls
-      --without-openssl
+      --without-appletls
+      --with-openssl
```

Comment:
Use `if OS.mac?` and `if OS.linux?` as appropriate. Linuxbrew supports both Mac and Linux. (Homebrew supports only Mac).